src/rate.rs

```diff
@@ -627,7 +627,7 @@ impl RCState {
     let reservoir_frame_delay = if maybe_reservoir_frame_delay.is_some() {
       maybe_reservoir_frame_delay.unwrap().max(12)
     } else {
-      ((max_key_frame_interval.checked_mul(3)?) >> 1).min(240)
+      ((max_key_frame_interval.checked_mul(3)?) >> 1).min(240).max(12)
```

Comment:
``` rust
let reservoir_frame_delay = if let Some(frd) = maybe_reservoir_frame_delay {
  frd
} else {
  ((max_key_frame_interval.checked_mul(3)?) >> 1).min(240)
}.max(12);
```

maybe?

Reply:
Didn't know you could do a `{}.func()`. Will change.